rca/static_src/sass/abstracts/_mixins.scss

```diff
@@ -148,12 +148,20 @@
             width: 0;
         }
     }
+}
+
+@mixin underline-hover-target-reset {
+    &::after {
+        width: 100%;
+        left: 0;
+    }
+}
+
+@mixin underline-hover() {
```

Comment:
This might be a bit hard to review but I basically just split this existing mixin into two separate mixins, so I could reuse the same effect but in the case where we want the whole block to be hover-able, but only one of its components to show the effect.

Reply:
Yup, that makes sense